app/services/dwp/benefit_check_service.rb

```diff
@@ -1,10 +1,71 @@
 module DWP
   class BenefitCheckService
+    BENEFIT_CHECKER_NAMESPACE = 'https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check'.freeze
+    USE_MOCK = ActiveModel::Type::Boolean.new.cast(ENV.fetch('BC_USE_DEV_MOCK', nil))
+    REQUEST_TIMEOUT = 30.seconds
+
+    class ApiError < StandardError
+      include Nesty::NestedError
```

Comment:
Do we need this include?

Reply:
What is the benefit of this gem over native errors? Is it a dependency that we really need? I don't think so given that it has three contributors/maintainers and was last updated in 2013... so I'm going to make a call here and say that we don't use this. Please use Ruby native errors.

https://github.com/skorks/nesty for reference